packages/clay-navigation/src/Breadcrumb.tsx

```diff
@@ -0,0 +1,122 @@
+/**
+ * © 2019 Liferay, Inc. <https://liferay.com>
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import classNames from 'classnames';
+import React from 'react';
+import warning from 'warning';
+import {BreadcrumbEllipsis} from './BreadcrumbEllipsis';
+import {BreadcrumbItem, IBreadcrumbItem} from './BreadcrumbItem';
+
+interface IProps extends React.HTMLAttributes<HTMLOListElement> {
+	ellipsisBuffer?: number;
```

Comment:
We will need JSDocs for these props since this is the public facing API

Reply:
Just one comment would not be enough?

Reply:
Added 
```javascript
/**
 * The number of Breadcrumb Items to show on each side of the active Breadcrumb Item before
 * using an ellipsis dropdown.
 */
```

It's okay?